components/domains/register.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { TextField } from "@mui/material";
```

Comment:
WE DON'T USE REGISTER ANYMORE BUT REGISTER V2, sorry it's confusing. 

You can delete register.tsx